resources/js/processes/modeler/components/inspector/TaskAssignment.vue

```diff
@@ -186,7 +186,7 @@
         return this.typeAssignmentExpression === "group";
       },
       specialAssignmentsListGetter () {
-        const value = _.get(this.node, "assignmentRules");
+        const value = _.get(this.node, "assignmentRules", '[]');
         return JSON.parse(value) || [];
```

Comment:
Is the or statement needed on this line now? Since we have a default value on line 189 can we remove the duplication on this line?

Reply:
Removing the `|| []` throws the same error again.

Reply:
I think the `_.get` on line  189 is redundant. What if you change it to:
```js
const value = this.node.get('assignmentRules') || '[]';
```

Do you then still need the `|| []`?

According to [the docs](https://lodash.com/docs/4.17.15#get),`_.get` will only fall back to the default value if it resolves to `undefined`, but the code I suggested above will fall back to the `'[]'` value if it's an empty string or `null` as well.

Reply:
`const value = this.node.get('assignmentRules') || '[]';` still produces the same error.